contracts/test/Fallback.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.11;
+
+import "../Libraries/Events.sol";
+
+contract Fallback is Events {
+    fallback() external payable {
+        if (msg.value > 0) emit LogUint(msg.value);
+        if (msg.data.length > 0) emit LogBytes(msg.data);
+    }
+
+    function fallback(bytes calldata data) external payable {
```

Comment:
This non-standard "fallback" function is to simulate a fallback with `msg.data`? Is that the purpose of this?

Reply:
If this non-standard "fallback" would be expected by the VM, then the targets would need to be compliant with this type of "fallback", which would not be the case for most deployed protocols.